src/main/java/io/cnaik/service/CommonUtil.java

```diff
@@ -66,7 +68,24 @@ public void send() {
         if(false) {
             json = responseMessageUtil.createCardMessage();
         } else {
-            json = "{ \"text\": \"" + responseMessageUtil.createTextMessage() + "\"}";
+            class Response {
```

Comment:
@yshingai I suggest you to create a new file for this class. Probably in the package `io.cnaik.model.google`.